cli/src/pcluster/config/cluster_config.py

```diff
@@ -1368,12 +1372,18 @@ def _register_storage_validators(self):
                         existing_fsx.add(storage.file_system_id)
                     else:
                         new_storage_count["fsx"] += 1
+                    self._register_validator(FsxLustreRegionValidator, region=self.region)
```

Comment:
what about generalizing the validator: `FeatureRegionValidator` and pass 2 parameters, the feature and the region, so: 
```python
self._register_validator(FeatureRegionValidator, feature=Feature.FSX_LUSTRE, region=self.region)
```

This will permit to add more feature validators or to remove them without the need to create new code.

Reply:
This corresponds to what you're doing with the utility function.
Then this permits to disable the region validators at once, by disabling the FeatureRegionValidator.

Reply:
I like it and I was thinking to do that since the beginning to reduce code duplication.
The only reason why I did not do that is consistency with the way we developed other validators: we have always preferred to decouple them and since validator names are visible to customers we have always preferred to return specific names to do ad-hoc overrides.

Nevertheless, I'm re.thinking about it thanks to your comment and I think that there is no reason to do an ad hoc override of this type of validators, so we can do the refactoring you want.

Reply:
I'd say we can start simple, with a single class, then in future we can decide if we want to split into multiple validators.